lib/rack/deflater.rb

```diff
@@ -66,7 +66,13 @@ def call(env)
         headers['content-encoding'] = "gzip"
         headers.delete(CONTENT_LENGTH)
         mtime = headers["last-modified"]
-        mtime = Time.httpdate(mtime).to_i if mtime
+        if mtime
+          begin
+            mtime = Time.httpdate(mtime).to_i
+          rescue ArgumentError
+            mtime = nil
+          end
```

Comment:
If we are going to do this (which I'm slightly against - isn't it an application bug that needs to be fixed?), we should at least log the error, otherwise it's silently ignoring it, which I'm strongly against.

Reply:
@ioquatix Ok, I'm with you in that point. The tracking error could be help the developer to discover where they code is wrong. But without raise, otherwise the fix doesnt make sense